lisa_commands/commands.py

```diff
@@ -330,25 +336,83 @@ def manga_me(query, _slack_event):
     """
     return request.anime_news_network_search('manga', query)
 
-def praise(query, _slack_event):
+def my_karma(_query, slack_event):
+    """
+    _query - query str (unused for this function)
+    slack_event - A dict of slack event information(unused for this function)
+
+    Returns the users karma:
+    """
+    text = random.choice([
+        'Sure thing! Your karma is: ',
+        "Let's see. Your current karma is: ",
+        'I have your karma at: ',
+        "Looks like your karma is: ",
+    ])
+
+    return text + request.dynamodb_query(
+        'karma_scores',
+        {'user': {'S': '<@' + slack_event['user'] + '>'}}
+    )['Item']['karma']['N']
+    
+def praise(query, slack_event):
     """
     query - query str
     slack_event - A dict of slack event information(unused for this function)
```

Comment:
```suggestion
    slack_event - A dict of slack event information
```
Since it's being used now